app/controllers/authentication.rb

```diff
@@ -14,14 +14,14 @@ def sign_out
 
   def authenticate_user!
     if !current_user
-      redirect_to root_url, :alert => 'You need to sign in for access to this page.'
+      redirect_to root_url, alert: 'You need to sign in for access to this page.'
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.